### ISSUE TITLE
Add slurm job submission for WPS, real.exe, and WRF jobs in workflow

### ISSUE THREAD

**Issue by mkavulich** [NCAR/wrfcloud#74]
## Describe the Enhancement ##
Currently all executables are run serially on the head node. We should introduce the capability to run some/all tasks as a submitted job

### Time Estimate ###
8 hours

### Sub-Issues ###
Consider breaking the enhancement down into sub-issues.
- [ ] Job submission for WRF parallel jobs
- [ ] Use generic function for job submission for any task
- [ ] Job monitoring so multiple tasks can be run in parallel (e.g. post jobs kick off as soon as output is available, while WRF is still running)
  - This is not required for initial release

### Relevant Deadlines ###
*September 30*

### Assignee ###
- [ ] Select **engineer(s)** or **no engineer** required
- [ ] Select **scientist(s)** or **no scientist** required

### Labels ###
- [ ] Select **component(s)**
- [ ] Select **priority**

### Projects and Milestone ###
- [ ] Select **Project**
- [ ] Select **Milestone** as the next official version or **Backlog of Development Ideas**

## Enhancement Checklist ##
- [ ] Complete the issue definition above, including the **Time Estimate** and **Funding Source**.
- [ ] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>/<Description>`
- [ ] Complete the development and test your changes.
- [ ] Add/update log messages for easier debugging.
- [ ] Add/update tests.
- [ ] Add/update documentation.
- [ ] Push local changes to GitHub.
- [ ] Submit a pull request to merge into **develop**.
Pull request: `feature <Issue Number> <Description>`
- [ ] Define the pull request metadata, as permissions allow.
Select: **Reviewer(s)**, **Project**, and **Development** issue
Select: **Milestone** as the next official version
- [ ] Iterate until the reviewer(s) accept and merge your changes.
- [ ] Delete your fork or branch.
- [ ] Close this issue.
